woocommerce/woocommerce-gutenberg-products-block
# Update previews and editor styles for cart and checkout blocks

This PR updates cart and checkout block previews.

- Fixes some display issues in the editor due to inheriting default styles
- Splits up the cart block into a block.js file to follow the patterns in checkout block and others
- Renamed `empty-cart` to `empty-cart-edit` so it's clear this is the editor side only
- Adds `isPreview` attribute to cart and checkout blocks so preview can be rendered
- Adds new SVGs for the 2 new previews
- Updates cart sample data to include correct images and totals data.

Fixes #1487

### Screenshots

Cart preview:
![Screenshot 2020-04-08 at 14 17 46](https://user-images.githubusercontent.com/90977/78788924-5bcd4c80-79a4-11ea-86dd-1c2f9d4e2b30.png)

Checkout preview:
![Screenshot 2020-04-08 at 14 17 58](https://user-images.githubusercontent.com/90977/78788912-57089880-79a4-11ea-91cd-40119b5b4f58.png)

Cart data:
![Screenshot 2020-04-08 at 14 21 37](https://user-images.githubusercontent.com/90977/78788883-4e17c700-79a4-11ea-984e-b14aac9a4c89.png)

### How to test the changes in this Pull Request:

1. Check the reviews by going to add a cart/checkout block via the editor
2. Smoke test cart editor and frontend cart to ensure the block still functions
